web/index.html

```diff
@@ -2,14 +2,37 @@
 <html>
 <head>
   <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+  <meta name="description" content="A new Flutter project.">
```

Comment:
This should probably have a better description.

Reply:
I have it fixed in next PR which also fixes favicons: https://github.com/flutter/gallery/pull/131